server/store.go

```diff
@@ -110,14 +110,21 @@ const (
 
 // StreamState is information about the given stream.
 type StreamState struct {
-	Msgs      uint64    `json:"messages"`
-	Bytes     uint64    `json:"bytes"`
-	FirstSeq  uint64    `json:"first_seq"`
-	FirstTime time.Time `json:"first_ts"`
-	LastSeq   uint64    `json:"last_seq"`
-	LastTime  time.Time `json:"last_ts"`
-	Deleted   []uint64  `json:"deleted,omitempty"`
-	Consumers int       `json:"consumer_count"`
+	Msgs      uint64          `json:"messages"`
+	Bytes     uint64          `json:"bytes"`
+	FirstSeq  uint64          `json:"first_seq"`
+	FirstTime time.Time       `json:"first_ts"`
+	LastSeq   uint64          `json:"last_seq"`
+	LastTime  time.Time       `json:"last_ts"`
+	Deleted   []uint64        `json:"deleted,omitempty"`
```

Comment:
@derekcollison hmm, thought we werent going to return this because it can be so much data? Less now since we dont MarshalIndent but still quite a lot I imagine in real world uses.  I also dont know what we'd do with this information, we cant undelete.  So maybe a counter of deleted messages could be good instead

Reply:
Interior deletes should be very rare, I think including them is important. Most deletes happen at the beginning dues to limits etc. They do not show up here since FirstSeq and FirstTime move.